tests/partitioninfo.spec.coffee

```diff
@@ -69,3 +70,135 @@ describe 'Partitioninfo:', ->
 					type: 131
 				}
 			])
+
+	describe '.getPartitions() disk 1 includeExtended = false', ->
+		it 'should list all partitions of a disk image', ->
+			promise = partitioninfo.getPartitions(DISK_PATH, false)
+			m.chai.expect(promise).to.eventually.become([
+				{
+					offset: 1048576
+					size: 1024
+					type: 131
+				}
+				{
+					offset: 1049600
+					size: 1024
+					type: 131
+				}
+				{
+					offset: 1050624
+					size: 1024
+					type: 131
+				}
+				{
+					offset: 2100224
+					size: 1024
+					type: 131
+				}
+				{
+					offset: 3149824
+					size: 1024
+					type: 131
+				}
+				{
+					offset: 4199424
+					size: 1024
+					type: 131
+				}
+				{
+					offset: 5249024
+					size: 1024
+					type: 131
+				}
+			])
+
+	describe '.get() disk 2', ->
+
+		it 'should be able to get the first logical partition of the extended one', ->
+			promise = partitioninfo.get(DISK2_PATH, 5)
+			m.chai.expect(promise).to.eventually.become
+				offset: 3145728
+				size: 1048576
+				type: 131
+
+		it 'should be able to get the second logical partition of the extended one', ->
+			promise = partitioninfo.get(DISK2_PATH, 6)
+			m.chai.expect(promise).to.eventually.become
+				offset: 5242880
+				size: 1048576
+				type: 131
+
+		it 'should return an error when asked for a non existing logical partition', ->
+			promise = partitioninfo.get(DISK2_PATH, 10)
+			m.chai.expect(promise).to.be.rejectedWith('Partition not found: 10.')
+
+		it 'should return an error when asked for a non existing primary partition', ->
+			promise = partitioninfo.get(DISK2_PATH, 4)
+			m.chai.expect(promise).to.be.rejectedWith('Partition not found: 4.')
+
+	describe '.getPartitions() disk 2', ->
+		it 'should list all partitions of a disk image', ->
+			promise = partitioninfo.getPartitions(DISK2_PATH)
+			m.chai.expect(promise).to.eventually.become([
+				{
+					'offset': 1048576
+					'size': 1048576
+					'type': 131
+				}
+				{
+					'offset': 2097152
+					'size': 7340032
+					'type': 5
+				}
+				{
+					'offset': 9437184
+					'size': 1048576
+					'type': 131
+				}
+				{
+					'offset': 3145728
+					'size': 1048576
+					'type': 131
+				}
+				{
+					'offset': 5242880
+					'size': 1048576
+					'type': 131
+				}
+				{
+					'offset': 7340032
+					'size': 1048576
+					'type': 131
+				}
+			])
+
+	describe '.getPartitions() disk 2 includeExtended = false', ->
+		it 'should list all partitions of a disk image', ->
```

Comment:
Minor nit, but I wonder if the 'it' line here should be different to the one on line 140?